### DISCUSSION TITLE
One package is missing from the list of installed packages

### DISCUSSION THREAD

**Discussion by hwfanatic** [microsoft/winget-cli#3813] [Q&A]
I have one package installed outside of winget that does not appear in winget list, but it does appear in the list of Installed Apps in Settings and Programs in Control Panel. 

I don't think this is such a big deal considering I will not be updating the package through winget, but I was wondering what could be the cause of this and how would I go about fixing it.

**Comment by denelon** [comment]
Without knowing what the package is, it's hard to know for sure.

My best guess is that the information in the registry either has characters WinGet doesn't understand, or some of the other data written by the installer is potentially malformed.
